Catalogue.md

```diff
@@ -0,0 +1,20 @@
+# Catalogue
+
+The TIDAL Catalogue module is a wrapper around our [Catalogue API](https://developer.tidal.com/reference/web-api?spec=catalogue).
```

Comment:
This links to our old catalogue API - it should link to the JSON:API one.

Reply:
Nah, not the OAS. At the top of api ref you have a drop down to select different doimains. We will deprecate the "V1" shortly and only have JSON API ones. So you should link to the "new" catalogue: https://developer.tidal.com/reference/web-api?spec=catalogue-v2&ref=get-albums-v2

Reply:
Got it, thanks! Will do!